tests/openssh.rs

```diff
@@ -25,9 +27,25 @@ fn loopback() -> IpAddr {
     "127.0.0.1".parse().unwrap()
 }
 
-async fn session_builder_connect(builder: SessionBuilder, addr: &str) -> Vec<Session> {
+fn get_known_hosts_path() -> &'static Path {
+    lazy_static! {
```

Comment:
Lazy static definitely seems like overkill here! It's testing code, and accessing env isn't all that expensive. So let's just make this a normal function

Reply:
Done